central/alert/datastore/internal/store/postgres/bench_test.go

```diff
@@ -40,21 +40,23 @@ func BenchmarkMany(b *testing.B) {
 
```

Comment:
I think, we need a `b.ResetTimer()` here.

Reply:
`b.Loop` takes care of this

> Loop resets the benchmark timer the first time it is called in a benchmark,
> so any setup performed prior to starting the benchmark loop does not count
> toward the benchmark measurement. Likewise, when it returns false, it stops
> the timer so cleanup code is not measured.

https://cs.opensource.google/go/go/+/refs/tags/go1.25.3:src/testing/benchmark.go;l=475-478
https://go.dev/blog/testing-b-loop#old-benchmark-loop-problems

Reply:
- https://github.com/go-critic/go-critic/issues/1513